external/webserver/http_client.c

```diff
@@ -950,17 +952,21 @@ static int http_send_chunk_buffer(struct http_client_t *client, char *buf)
 	return 0;
 }
 
-static int http_send_chunk(struct http_client_t *client, char *buf, int buflen, const char *body, bool is_last_msg)
+static int http_send_chunk(struct http_client_t *client, char *buf, int buflen,
+					const char *body, int body_len, bool is_last_msg)
```

Comment:
This multi-line function definition can be taken care of in one line itself by using separators please.

Reply:
not required